GCP/gcp-project-switcher.1d.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Metadata allows your plugin to show up in the app, and website.
+#
+#  <xbar.title>GCP Project Switcher</xbar.title>
+#  <xbar.version>v1.0</xbar.version>
+#  <xbar.author>Daniel Palma</xbar.author>
+#  <xbar.author.github>danthelion</xbar.author.github>
+#  <xbar.desc>See current active GCP project and easily switch between others.</xbar.desc>
```

Comment:
Thanks for the plugin - this is one I'll use. Any chance you could add an image?